.gitignore

```diff
@@ -4,3 +4,5 @@
 # Environment Variables
 *.env
 !example.env
+client/client.iml
```

Comment:
These should be clearly labeled with a comment
(see the rest of the file for examples)